### PR TITLE
[AV-1850] update config flags to match geth v1.10.20 changes

### PR DESCRIPTION
**Pull request by darioush**
merge with next public release of AvalancheGo

### CODE REVIEW

**Review by aaronbuchwald — APPROVED** [review]
LGTM - this should be merged with the next coreth/avalanchego release.

File: docs/nodes/maintain/chain-config-flags.md [hunk 1]
```diff
@@ -122,20 +122,47 @@ Specifies the directory for the Admin API to use to store CPU/Mem/Lock Profiles.
...
+
+
 :::note
 
 If you populate this field, it will override the defaults so you must include every service you wish to enable.
 
 :::
 
-#### `eth`:
+#### `public-eth` (deprecated) or `eth`:
```
Comment (aaronbuchwald) [inline docs/nodes/maintain/chain-config-flags.md:158]: @yulin-dong how do we normally handle deprecation notices like this?

It seems strange to keep the deprecated name in the header here. Should we move the old name and deprecation notice into the body and say something like:

```
The API name`public-eth` is deprecated as of v1.7.15, and the APIs previously under this name have been migrated to `eth`.
```

Comment (yulin-dong) [reply docs/nodes/maintain/chain-config-flags.md:158]: Yeah, I like your suggestion better, @aaronbuchwald

**Review by yulin-dong — APPROVED** [review]

**Comment by yulin-dong** [comment]
I will merge it to branch 1.7.5 which will be merged to master when we are ready.

**Pull request merged**